src/routes/playlists/tripleS/+page.svelte

```diff
@@ -0,0 +1,662 @@
+<script lang="ts">
+  import AlbumPlaylistCard from './AlbumPlaylistCard.svelte'
+  import MemberProfileCardSwag from './MemberProfileCardSwag.svelte'
+  import OneClickDownloadButton from '$lib/OneClickDownloadButton.svelte'
+  import ZipDownloadButton from '$lib/ZipDownloadButton.svelte'
+  import MetaHead from '$lib/MetaHead.svelte'
+  import { audioPlayer } from '$lib/audio-player'
+
+  let nextSToClick = 1
+  const clickMember = (sNumber: number) => {
+    if (sNumber === nextSToClick) {
+      nextSToClick++
+    }
+
+    console.log('nextSToClick', nextSToClick)
```

Comment:
Console log

Reply:
My specialty. I even removed another one there, but still forgot this one. :laughing: